### ISSUE TITLE
False negatives due to timestamps in CRAN checks

### ISSUE THREAD

**Issue by lionel-** [r-lib/rcmdcheck#128]
Comparing against CRAN checks causes false failures because they include timestamps:

```r
[[1]]
── R CMD check comparison ───────────────────────────── HyperbolicDist 0.6-2 / 0.6-2 ────
Status: BROKEN

── Fixed

✔ checking R code for possible problems ... [21s/24s] NOTE

── Newly failing

✖ checking R code for possible problems ... NOTE


[[2]]
── R CMD check comparison ───────────────────────────────── GlobalDeviance 0.4 / 0.4 ────
Status: BROKEN

── Fixed

✔ checking R code for possible problems ... [6s/8s] NOTE

── Still failing

✖ checking DESCRIPTION meta-information ... NOTE

── Newly failing

✖ checking R code for possible problems ... NOTE
```

Perhaps these could be safely stripped with a simple regexp?

**Comment by gaborcsardi** [comment]
Yeah, or we could have an option to not include them at all. I am actually surprised that they are included.

**Comment by lionel-** [comment]
This seems to solve it: https://github.com/r-lib/rcmdcheck/commit/b5f1fbd7746f675bf475e3a163c55a9042871922

**Comment by gaborcsardi** [comment]
Right, we should probably remove it from the local checks as well, in case.
